packages/lists/src/components/FixedList/FixedList.tsx

```diff
@@ -0,0 +1,164 @@
+import * as React from 'react';
+import { IViewportState, ScrollDirection, Axis } from '../Viewport/Viewport';
+
+export interface IFixedListProps {
```

Comment:
separate the props into a types file. It helps to keep them separate as we can auto doc it.